### DISCUSSION TITLE
Can we add URL in HL7 2.3 MDM Message?

### DISCUSSION THREAD

**Discussion by BforAzure2** [nextgenhealthcare/connect#4773] [Q&A] [answered]
Hello guys, we have a problem where we want to send URL in HL7 Message. Can we add URL? If so, what will be the appropriate segment to add that. It is a URL of a document in MDM message but also required in other message types like results/orders

**Comment by tippenring** [comment, marked as answer]
We do this with AthenaPractice (formerly Centricity Practice Solution). It is in an OBX segment.

`OBX|0001|ST|IMAGE_REF||http://docserver/v1/document/573a880546ac4.pdf`

**Reply by BforAzure2** [reply]
@tippenring Thank you for your reply. Appreciated!
